src/test/java/seedu/placebook/logic/commands/AddAppCommandTest.java

```diff
@@ -28,8 +29,11 @@
 import seedu.placebook.model.schedule.Appointment;
 import seedu.placebook.testutil.AppointmentBuilder;
 import seedu.placebook.testutil.PersonBuilder;
+import seedu.placebook.ui.Ui;
 
 public class AddAppCommandTest {
+    // default positive confirmation ui. This will not affect AddAppCommand
+    private static final Ui uiStub = UiStubFactory.getUiStub(true);
```

Comment:
Similarly should be named UI_STUB.

Also since none of the methods in the uiStub should be called here, maybe it is better to make another class in UiStubFactory for notCalledConfirmationUi, where all the methods would throw an exception for better testing.

Then instead of using a boolean for getUiStub can use another Enum.

Reply:
Im looking at the other commands, and since uiStubs have to be used everywhere now, is it possible to create the instances of UiStub in testUtil or somewhere similar to have better readability?

Reply:
I think that instance is in the Factory, I am just getting them so that I dont have to write the long name later. Is this ok?

Reply:
Oh ok should be fine